native/cocos/bindings/jswrapper/v8/Object.h

```diff
@@ -152,6 +152,7 @@ class Object final : public RefCounter {
      *  @note The return value (non-null) has to be released manually.
      */
     static Object *createJSONObject(const ccstd::string &jsonStr);
+    static Object *createJSONObject(std::u16string &jsonStr);
```

Comment:
It is better to add a comment saying that, jsonStr is moved in `createJSONObject `.

Reply:
Updated.